samples/cpp/text_generation/README.md

```diff
@@ -1,6 +1,17 @@
-# Text generation C++ greedy_causal_lm that supports most popular models like LLaMA 3
 
-This example showcases inference of text-generation Large Language Models (LLMs): `chatglm`, `LLaMA`, `Qwen` and other models with the same signature. The application doesn't have many configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU. The sample fearures `ov::genai::LLMPipeline` and configures it to run the simplest deterministic greedy sampling algorithm. There is also a Jupyter [notebook](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/llm-chatbot) which provides an example of LLM-powered Chatbot in Python.
+# OpenVINO AI Text Generation Samples
+
+These samples showcase the use of OpenVINO's inference capabilities for text generation tasks, including different decoding strategies such as beam search, multinomial sampling, and speculative decoding. Each sample has a specific focus and demonstrates a unique aspect of text generation.
+The applications don't have many configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU.
+There is also a Jupyter [notebook](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/llm-chatbot) that provides an example of LLM-powered text generation in Python.
+
+## Table of Contents
+1. [Download and Convert the Model and Tokenizers](#download-and-convert-the-model-and-tokenizers)
+2. [Running the Samples](#running-the-samples)
+3. [Using encrypted models](#using-encrypted-models)
```

Comment:
this section is missed.

BTW, it not a dedicated section, it's also just a sample as others.